contracts/ColonyTask.sol

```diff
@@ -288,18 +264,39 @@ contract ColonyTask is ColonyStorage, DSMath {
   function finalizeTask(uint256 _id) public
   auth
   taskExists(_id)
-  taskWorkRatingsAssigned(_id)
+  taskWorkRatingsClosed(_id)
   taskNotFinalized(_id)
   {
     Task storage task = tasks[_id];
     IColonyNetwork colonyNetworkContract = IColonyNetwork(colonyNetworkAddress);
 
+    int8[3] memory ratingMultiplicator = [-10, 10, 15];
+
     for (uint8 roleId = 0; roleId <= 2; roleId++) {
       uint payout = task.payouts[roleId][token];
       Role storage role = task.roles[roleId];
-      uint8 rating = (roleId == EVALUATOR) ? 50 : role.rating;
-      int divider = (roleId == WORKER) ? 30 : 50;
-      int reputation = SafeMath.mulInt(int(payout), (int(rating)*2 - 50)) / divider;
+
+      uint8 rating = (roleId == EVALUATOR) ? 3 : role.rating;
+      int reputation = SafeMath.mulInt(int(payout), (int(ratingMultiplicator[rating - 1]))) / 10;
+
+      // If rating is 0 at this point, we assign the highest possible rating and give
+      // a reputation penalty to the user who didn't submit a rating on time.
+      if (rating == 0 && roleId == MANAGER) {
```

Comment:
The logic that follows here isn't very clear to me and I think there's an issue in that you are penalising the skill reputation of the worker (if they haven't submitted) but then you are also awarding them the full reputation (line 301).
The rules are: everyone earns domain reputation, worker earns skill reputation. Evaluator and worker get penalised on domain and domain and skill reputation respectively if they don't rate on time.